pkg/registry/rbac/clusterrolebinding/policybased/storage.go

```diff
@@ -63,6 +66,12 @@ func (s *Storage) Update(ctx genericapirequest.Context, name string, obj rest.Up
 	nonEscalatingInfo := rest.WrapUpdatedObjectInfo(obj, func(ctx genericapirequest.Context, obj runtime.Object, oldObj runtime.Object) (runtime.Object, error) {
 		clusterRoleBinding := obj.(*rbac.ClusterRoleBinding)
 
+		// if we're explicitly authorized to bind this clusterrole, return
+		if canBind, _, _ := rbacregistry.AuthorizeBinding(ctx, clusterRoleBinding.RoleRef, clusterRoleBinding.Namespace, s.authorizer); canBind {
```

Comment:
I'm fine w/ not logging the `canBind == false` response. Can we at least log in case of an error? Or do some authorizers not respect the `allowed, message, err` format?